sirepo_bluesky/tests/test_sirepo_det.py

```diff
@@ -0,0 +1,52 @@
+import vcr
+import os
+
+from sirepo_bluesky.sirepo_detector import SirepoDetector
+import bluesky.plans as bp
+import sirepo_bluesky.tests
+
+cassette_location = os.path.join(os.path.dirname(sirepo_bluesky.tests.__file__), 'vcr_cassettes')
+
+
+@vcr.use_cassette(f'{cassette_location}/test_sirepo_detector.yml')
+def test_sirepo_detector(RE, db, tmpdir):
+    import datetime
+    from ophyd.utils import make_dir_tree
+
+    RE.subscribe(db.insert)
+
+    root_dir = '/tmp/data'
```

Comment:
Did `tmpdir` fail with some weird errors?

Reply:
It did fail because the directory couldn't be found. I think the SirepoDetector class creates the directory for DataBroker within it, so that's something we can probably change so it acts like the flyer class